### PR TITLE
Replaced choose with select

### PR DESCRIPTION
**Pull request by JoshuaAdemola**

### CODE REVIEW

**Review by MWindel — APPROVED** [review]

File: content/users-guide/administration-bundle/managing-users.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ The user management feature allows you to manage the users within your tenant. W
 - Create users.
 - Assign usernames and set passwords.
 - Store user details.
-- Choose login options.
+- Select login options.
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-users.md:18]: I would use "Specify" in this case

File: content/users-guide/device-management-bundle/connecting-devices.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ Devices can be connected to your {{< product-c8y-iot >}} account in different wa
 
 ### To register devices
 
-To register devices, you may choose one of the following options:
+To register devices, you may select one of the following options:
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/connecting-devices.md:44]: may select > can select

File: content/users-guide/enterprise-tenant-bundle/ms-data-broker.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ aliases:
...
 
-> **Important:** At this time the {{< product-c8y-iot >}} Messaging Service and the microservice-based data broker are available on a limited trial basis, and not yet enabled in all {{< product-c8y-iot >}} public cloud environments. Please contact [product support](/welcome/contacting-support/) to inquire about using these capabilities in your {{< product-c8y-iot >}} environment. The original data broker will continue to operate alongside the microservice-based data broker for the time being. Users can choose which data broker to use on a per-tenant basis.
+> **Important:** At this time the {{< product-c8y-iot >}} Messaging Service and the microservice-based data broker are available on a limited trial basis, and not yet enabled in all {{< product-c8y-iot >}} public cloud environments. Please contact [product support](/welcome/contacting-support/) to inquire about using these capabilities in your {{< product-c8y-iot >}} environment. The original data broker will continue to operate alongside the microservice-based data broker for the time being. Users can select which data broker to use on a per-tenant basis.
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/ms-data-broker.md:11]: In this case, it should stay "choose".

File: content/web/tutorials-bundle/extend-an-existing-application.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ The hybrid mode, however, comes with some limitations we will cover later in thi
...
  - [@c8y/cli](/web/development-tools/#c8y-cli): Is our pre-caved way that integrates well with our tooling but most likely will not allow special cases.
 
- So in total, there are three possibilities to start with the Web SDK: Extending an existing hybrid app, building a pure Angular app with Angular CLI or building it with @c8y/cli. Which one to choose heavily depends on the application you want to build. For example, if you want an application that just follows the look&feel of the platform but want to use special dependencies for certain scenarios (for example Material-Framework), you are best with the pure Angular CLI solution.
+ So in total, there are three possibilities to start with the Web SDK: Extending an existing hybrid app, building a pure Angular app with Angular CLI or building it with @c8y/cli. Which one to select heavily depends on the application you want to build. For example, if you want an application that just follows the look&feel of the platform but want to use special dependencies for certain scenarios (for example Material-Framework), you are best with the pure Angular CLI solution.
```
Comment (BeateRixen) [inline content/web/tutorials-bundle/extend-an-existing-application.md:22]: In this case, it should stay "choose".

File: content/microservice-sdk/http-bundle/microservice-nodejs.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ title: Node.js microservice
 layout: redirect
 ---
 
-{{< product-c8y-iot >}} provides SDKs for developing microservices using C# or Java. Nevertheless, you are free to choose the tech-stack of your preference to develop a microservice as long as it fulfills the [general requirements](/microservice-sdk/concept/#requirements).
+{{< product-c8y-iot >}} provides SDKs for developing microservices using C# or Java. Nevertheless, you are free to select the tech-stack of your preference to develop a microservice as long as it fulfills the [general requirements](/microservice-sdk/concept/#requirements).
```
Comment (BeateRixen) [inline content/microservice-sdk/http-bundle/microservice-nodejs.md:7]: In this case, it should stay "choose".

File: content/protocol-integration/opcua-bundle/gateway-register.md [hunk 6]
```diff
@@ -225,7 +225,7 @@ gateway:
     lifetimeCount: 600
     # The notificationBufferSize defines how many monitored item values should be buffered to receive
     # subscription notification data from the OPC UA server. The subscription reporting rate (publish interval)  
-    # and the volume of sampling data should be taken into account to choose a suitable buffer size.
+    # and the volume of sampling data should be taken into account to select a suitable buffer size.
```
Comment (BeateRixen) [inline content/protocol-integration/opcua-bundle/gateway-register.md:228]: This is part of code comment, so we must leave it as is.

File: content/users-guide/administration-bundle/managing-users.md (hunk 1 above)
Comment (JoshuaAdemola) [reply content/users-guide/administration-bundle/managing-users.md:18]: Fixed

File: content/users-guide/device-management-bundle/connecting-devices.md (hunk 2 above)
Comment (JoshuaAdemola) [reply content/users-guide/device-management-bundle/connecting-devices.md:44]: Fixed

File: content/users-guide/enterprise-tenant-bundle/ms-data-broker.md (hunk 3 above)
Comment (JoshuaAdemola) [reply content/users-guide/enterprise-tenant-bundle/ms-data-broker.md:11]: Fixed

File: content/web/tutorials-bundle/extend-an-existing-application.md (hunk 4 above)
Comment (JoshuaAdemola) [reply content/web/tutorials-bundle/extend-an-existing-application.md:22]: Fixed

File: content/microservice-sdk/http-bundle/microservice-nodejs.md (hunk 5 above)
Comment (JoshuaAdemola) [reply content/microservice-sdk/http-bundle/microservice-nodejs.md:7]: Fixed

File: content/protocol-integration/opcua-bundle/gateway-register.md (hunk 6 above)
Comment (JoshuaAdemola) [reply content/protocol-integration/opcua-bundle/gateway-register.md:228]: Fixed

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**